04 - Guides, Workflows, & Courses/Community Talks/Metadata Group Discussion.md

```diff
@@ -8,4 +8,10 @@ publish: true
 
 # Metadata Group Discussion
 
-This [[Obsidian Community Talks|community talk]] was a group discussion about ways people use and manage their [[YAML frontmatter]]. 
\ No newline at end of file
+This [[Obsidian Community Talks|community talk]] was a group discussion about ways people use and manage their [[YAML frontmatter]]. 
```

Comment:
What's happening here with the line?

Reply:
There was no end-of-line character in the original file.

This original meant that the `%% Hub footer: Please don't edit anything below this line %%` comment ran right up against the end of the content.

I made the script first effectively:

1. add the missing end-of-line character at the end of the file, and then
2. add the footer.

Git shows this up as an edit on the last line of the file...